### DISCUSSION TITLE
Replace clock icon with bell icon

### DISCUSSION THREAD

**Discussion by jmlich** [InfiniTimeOrg/InfiniTime#1883] [Ideas]
I would like to replace clock icon (0xF017) with bell icon (0xF0F3). In my opinion the bell icon better describe that it is actually alarm.

https://fontawesome.com/icons/clock?f=classic&s=solid 
https://fontawesome.com/icons/bell?f=classic&s=solid

I wanted to prepare pull request and test it on my own, but I wasn't able to make it work. Maybe someone can tell me what I am doing wrong.

I was able to verify that attached fontawesome.ttf contain symbol:

```
convert -background black -fill white -font FontAwesome5-Solid+Brands+Regular.woff -pointsize 80 -gravity center label:"$(printf "\uf017")\n$(printf "\uf0f3")"  navigation0.png
```
![navigation0](https://github.com/InfiniTimeOrg/InfiniTime/assets/6171637/567fe5e6-28f3-4604-bdcc-a8f40a8973fc)

But when I replaced 0xf017 with 0xf0f3 in fonts.json the icon disappeared in InfiniSim.
https://github.com/InfiniTimeOrg/InfiniTime/blob/eac460f03094a357c679c5dede7295aa8b2e1fb1/src/displayapp/fonts/fonts.json#L10C63-L10C69

**Comment by FintasticMan** [comment]
You need to not only update fonts.json, but also the UTF-8 code in `src/displayapp/screens/Symbols.h`. See https://github.com/InfiniTimeOrg/InfiniTime/tree/main/src/displayapp/fonts#readme.

**Reply by jmlich** [reply]
I see, thanks for explanation. I overlooked that information in documentation. It seems other bell icon is used also for "loud profile" so using bell might be confusing. It seems I must shuffle more icons to reach that target. I will open new discussion when I have some proposal how it should look like.
